main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -85,7 +85,8 @@ object Main {
     var options = Options(
       lib = cmdOpts.xlib,
       debug = cmdOpts.xdebug,
-      documentor = cmdOpts.documentor,
+      documentorJson = cmdOpts.documentorJson,
```

Comment:
The JsonDocumenter is out-of-date. I don't want to delete it-- since I may restore it later.

But instead of having two cmd options let us just keep our single doc command.

We can then use it to trigger *both* documentors. Makes sense?